src/tests/components/NewArticleForm.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import NewArticleForm from '../../components/Articles/NewArticleForm';
+
+describe('NewArticleForm component', () => {
+  const mockStore = configureMockStore();
+  let wrapper;
+  const handleSubmit = jest.fn();
+  const resetForm = jest.fn();
+  const handleChange = jest.fn();
+  const handleEditorChange = jest.fn();
+  const title = '';
+  const description = '';
+  const body = '';
+  const loading = false;
+
+  beforeEach(() => {
+    mockStore({});
+    wrapper = shallow(<NewArticleForm
+      handleSubmit={handleSubmit}
+      resetForm={resetForm}
+      handleChange={handleChange}
+      handleEditorChange={handleEditorChange}
+      body={body}
+      title={title}
+      description={description}
+      loading={loading}
+    />);
+  });
+
+
+  it('should render correctly', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should call resetForm when the clear button is clicked', () => {
+    wrapper.find('#clear-button').simulate('click', { preventDefault() {} });
+    expect(resetForm).toBeCalled();
+  });
+
+  it('should call handleSubmit when the form is submitted', () => {
+    wrapper.find('#add-article-form').simulate('submit');
+    expect(handleSubmit).toBeCalled();
+  });
+
+  it('should call handleChange when the there is an input in title/description', () => {
+    wrapper.find('#title').simulate('change');
+    expect(handleChange).toBeCalled();
+  });
+
+  it('should call handleEditorChange when the there is an input in text editor', () => {
```

Comment:
Consider changing `when the there` on line 47 and 52 to make the test for meaningful.

Reply:
@kakaemma lines 47 and 52 have been updated with more descriptive comment